app/src/main/java/org/simple/clinic/di/DateFormatterModule.kt

```diff
@@ -23,7 +23,7 @@ class DateFormatterModule {
 
   @Provides
   @Named("exact_date")
-  fun providesTimeFormatterForRecentPatientsHeader(locale: Locale): DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM, yyyy", locale)
+  fun providesTimeFormatterForFullReadableDate(locale: Locale): DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM, yyyy", locale)
```

Comment:
Can we rename the method name?
```suggestion
  fun providesDateFormatterForFullReadableDate(locale: Locale): DateTimeFormatter = DateTimeFormatter.ofPattern("d MMMM, yyyy", locale)
```

Reply:
Done.